packages/contracts/src/2.0.0/tokens/EtherToken/WETH9.sol

```diff
@@ -22,10 +22,10 @@ contract WETH9 {
     string public symbol   = "WETH";
     uint8  public decimals = 18;
 
-    event  Approval(address indexed src, address indexed guy, uint wad);
-    event  Transfer(address indexed src, address indexed dst, uint wad);
-    event  Deposit(address indexed dst, uint wad);
-    event  Withdrawal(address indexed src, uint wad);
+    event  Approval(address indexed _owner, address indexed _spender, uint _value);
```

Comment:
Rather than modifying these events, can we just use `IEtherToken.sol`? (looks like we need to add the `Deposit` and `Withdrawal` events to that, which should be there anyways)

Reply:
Discussed offline, will be addressed in the future.

Reply:
Discussed offline, will be addressed in a future PR.